Formula/jobber.rb

```diff
@@ -0,0 +1,66 @@
+class Jobber < Formula
+  desc "Alternative to cron, with better status-reporting and error-handling"
+  homepage "https://dshearer.github.io/jobber/"
+  url "https://github.com/dshearer/jobber/archive/v1.4.2.tar.gz"
+  sha256 "a7a0f3034d5177038e22c861e0beacce1a5b6de29f4cef41a50ee422132934b4"
+  head "https://github.com/dshearer/jobber.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "./configure", "--prefix=#{prefix}", "--libexecdir=#{libexec}", "--sysconfdir=#{etc}",
+      "--localstatedir=#{var}"
+    system "make", "install"
+  end
+
+  plist_options :startup => true
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Label</key>
+          <string>#{plist_name}</string>
+          <key>Program</key>
+          <string>#{libexec}/jobbermaster</string>
+          <key>OnDemand</key>
+          <false/>
+          <key>RunAtLoad</key>
+          <true/>
+          <key>KeepAlive</key>
+          <true/>
+          <key>StandardOutPath</key>
+          <string>/var/log/jobber.log</string>
+          <key>StandardErrorPath</key>
+          <string>/var/log/jobber.log</string>
+        </dict>
+      </plist>
+    EOS
+  end
+
+  test do
+    expected_output = "Hi!"
+
+    (testpath/".jobber").write <<~EOS
+      version: 1.4
+      jobs:
+        Test:
+          cmd: 'echo "#{expected_output}" > "#{testpath}/output"'
+          time: '*'
+    EOS
+
+    begin
+      pid = fork do
+        exec libexec/"jobberrunner", "#{testpath}/.jobber"
+      end
+      sleep 3
+    ensure
+      Process.kill("TERM", pid)
+      Process.wait(pid)
+    end
```

Comment:
It's easier like:

```suggestion
    fork do
      exec libexec/"jobberrunner", "#{testpath}/.jobber"
    end
    sleep 3
```

Reply:
Will brew ensure that the jobberrunner process is killed?

Reply:
I guess so!